CMakeLists.txt

```diff
@@ -68,6 +68,18 @@ function(determine_version OUTPUT_VARIABLE)
     message(FATAL_ERROR "Failed to parse version number: ${GIT_VERSION}")
   endif()
 
+  if(APPLE OR MSVC)
+    string(REGEX MATCH "\\+full$" FULL_SUFFIX_MATCH ${NEW_VERSION})
+    if(NOT FULL_SUFFIX_MATCH)
+      set(NEW_VERSION ${NEW_VERSION}.full)
```

Comment:
Shouldn't this be `+full` instead?

Reply:
No, the only two cases I'm expecting here are:
```
v0.7.0-dev.123+g456abc
v0.7.0+full
```
So we need to append `.full` to end up with dot-separated metadata.

Reply:
OK thanks